src/components/Map.tsx

```diff
@@ -22,9 +22,9 @@ const Map: React.FC = (): JSX.Element => {
        * Initialize application
        */
 
-      const globalApikey =
-        "AAPK66d0fa7f737142fcb780f9fd1c8155f04YBLPof_SmYB-dzJgafylrodNieETG6SVXdYeo75C5eFIt9NykPed6jRtTI_c01w";
-      esriConfig.apiKey = globalApikey;
+      const { REACT_APP_GLOBAL_API_KEY } = process.env;
+
+      esriConfig.apiKey = `${REACT_APP_GLOBAL_API_KEY}`;
```

Comment:
One more thing, line 27 can result in the apiKey being "undefined" if the API key isn't found. I guess its not a big deal since it will throw an error either way, but something like this might be more clear
```js
const { REACT_APP_GLOBAL_API_KEY } = process.env;

if (!REACT_APP_GLOBAL_API_KEY) {
  throw new Error("API Key not found");
}
esriConfig.apiKey = REACT_APP_GLOBAL_API_KEY;
```

Reply:
👍 error handling! Will update that - I always forget about error handling which is pretty bad.... LOL